### PR TITLE
Create law and test for torque

### PR DESCRIPTION
**Pull request by alesanter**

### CODE REVIEW

File: symplyphysics/laws/dynamics/torque_due_to_twisting_force.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+)
+
+# Description
+## Torque is a turning action on a body about a rotation axis due to a force.
+
+# Law: tau = F * r * sin(phi)
+## tau - torque
+## F - force exerted at the given point
+## r - distance to axis from the given point
+## phi - angle between the position vector and force vector
```
Comment (blackyblack) [inline symplyphysics/laws/dynamics/torque_due_to_twisting_force.py:19]: What is `position vector`?

**Review by blackyblack — APPROVED** [review]

File: symplyphysics/laws/dynamics/torque_due_to_twisting_force.py (hunk 1 above)
Comment (alesanter) [reply symplyphysics/laws/dynamics/torque_due_to_twisting_force.py:19]: In Russian terminology it's mostly known as radius vector, which is how it is sometimes referred to in English. In most English sources I found it to be called [position vector](https://www.sciencedirect.com/topics/mathematics/position-vector)

Comment (alesanter) [reply symplyphysics/laws/dynamics/torque_due_to_twisting_force.py:19]: Or do you mean the position vector of what thing I am referring to here?

Comment (blackyblack) [reply symplyphysics/laws/dynamics/torque_due_to_twisting_force.py:19]: Yes, add short explanation, how to find position vector.

**Review by blackyblack — APPROVED** [review]

**Pull request merged**